Gopkg.lock

```diff
@@ -332,8 +332,8 @@
 [[projects]]
   name = "github.com/spf13/pflag"
   packages = ["."]
-  revision = "e57e3eeb33f795204c1ca35f56c44f83227c6e66"
-  version = "v1.0.0"
+  revision = "9a97c102cda95a86cec2345a6f09f55a939babf5"
```

Comment:
if we need a particular version, it can be defined in `Gopkg.toml`, then the `Gopkg.lock` is updated by running `make vendor`. Before we merge this, how about if we merge #2121 first? It's just waiting on the CI.

Reply:
Yes, that will definitely work. i can rebase this once #2121 merges.